docs/docs/components/sort.md

```diff
@@ -0,0 +1,73 @@
+---
+id: sort
+title: Sort
+---
+
+`Sort` renders a list of possible sort by and sort order options in one single component.
```

Comment:
is the rendering in one component the only option from now on? because it is not very comfortable to use when building responsive pages

Reply:
No, you have `SortBy, `SortOrder`, or `Sort` for all together.